src/Benchmarks/FortunesEfMiddleware.cs

```diff
@@ -44,9 +44,7 @@ public FortunesEfMiddleware(RequestDelegate next)
 
         private static async Task<IEnumerable<Fortune>> LoadRows(ApplicationDbContext dbContext)
         {
-            var result = new List<Fortune>();
-
-            result.AddRange(await dbContext.Fortunue.ToListAsync());
+            var result = await dbContext.Fortune.ToListAsync();
```

Comment:
Does foreknowledge mean that the *test* mustn't pre-allocate, or that the "innocent" DB call can't pre-allocate? In neither case does the *test* pre-allocate anything here. (I.e. it's not telling EF that there are 2 rows and to prepare for that.)